backend/canvas_app_explorer/admin.py

```diff
@@ -14,7 +14,8 @@ class LtiToolAdmin(admin.ModelAdmin):
         'privacy_agreement',
         'support_resources',
         'canvas_placement',
-        'internal_notes'
+        'internal_notes',
+        'launch_url',
```

Comment:
`launch_url` kind of gets lost down at the end of the form. The tuple order controls the form order.

I'd consider moving it up the list, either in the same line as `canvas_id` (like how `logo_image` is grouped) or immediately after `canvas_id`. If the `help_text` was defined on both or one of these fields it would make it clearer how to define this for an admin.

Reply:
moved the help_text up, next to canvas_id